src/cls/rbd/cls_rbd.cc

```diff
@@ -138,9 +140,17 @@ cls_method_handle_t h_group_create;
 cls_method_handle_t h_group_dir_list;
 cls_method_handle_t h_group_dir_add;
 cls_method_handle_t h_group_dir_remove;
+cls_method_handle_t h_group_image_add;
+cls_method_handle_t h_group_image_remove;
+cls_method_handle_t h_group_image_list;
+cls_method_handle_t h_group_image_clean;
+cls_method_handle_t h_group_image_add_uplink;
```

Comment:
I would rename these three functions to ```image_add_group```, ```image_remove_group```, and ```image_get_group``` where the ```image_``` prefix denotes that it is an operation against the image header.